config/karma-docs.conf.js

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
@matthewrfindley  - For those file that were present before you on-boarded (such as this), I think it best tha I provide more comments [in those areas]. This will enable you to focus only on your additions and changes.